advice_for_future_giraf/PO_advice.md

```diff
@@ -0,0 +1,151 @@
+The other groups should not be present for these meetings as it is your responsibility to relay this information to the other groups.
+We arranged a meeting with Emil from Egebakken right after his initial presentation. 
+A transcription of this interview can be seen in the PO report of 2019.
+It is a good idea to get their phone numbers if possible, as it has been evident from our experience that communication through mail is often lacking.
+After this meeting you should have a good idea of what the customers want.
+You can then start setting goals for what should be completed in the upcoming sprint, as well as for the whole semester.
+
+As soon as you have the start and end dates for you sprints solidified, you should contact as many customers as you can with dates for usability testing. 
+We did this by sending a mail to every customer involved in the project.
+Remember to schedule usability testing after your releases so that you have a working program for the customers to test and evaluate.
+It is important that you ask the customers to confirm that they will participate in these tests.
+If they do not respond you should try and contact them to see if they simply forgot to reply.
+We did this by calling them on their phones or by calling their workplaces.
+
+It is very important to take notes and document all meetings with customers.
+You will receive a lot of questions from the different developer groups about how functionality should be made and how it should look.
+Therefore, it is always better to have written down exactly what the customers want so that you do not have to guess and then refactor later if you guessed wrong.
+An issue we faced was that we thought it would be sufficient for a guardian to copy activities one day at a time, but when we showed prototypes to the guardians they were very adamant on having the functionality be able to copy to multiple days at once.
+Another reason to document as much as possible, is that the users' feedback is the basis of all the user stories you are going to create.
+
+## Creating user stories
+As the PO group it is your responsibility to create user stories.
+User stories are created based on requests from the users.
+We structured user stories in the following way:
+- As a ... I would like ..., so that ...
+
+When creating a user story you should consider the amount of work that is needed for it to be completed, and whether or not it should be split up into multiple smaller user stories.
+We created user stories in the weekplanner repository on our GitHub page under the issue tab.
+This made it easy to organize as user stories are uniquely numbered and can be put into milestones.
+When creating a pull request it is easy to tag the user story so that it will be automatically closed when the pull request has been merged.
```

Comment:
Tell them how or refer them to a link that does that. We will add a "How to: Create a Pull Request" section where this will be described.

Reply:
If you're in doubt how you write anchors, look at this: https://gist.github.com/TobiasPalludan/5948533e97d260a04ea47e44ea7cc6b0

Reply:
I added a to-do in our Jira to update it when how to create a PR is added